app/Services/CategoryService.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Services;
+
+use App\Repositories\CategoryRepository;
+
+class CategoryService
+{
+    protected $categoryRepository;
+
+    public function __construct(CategoryRepository $categoryRepository)
+    {
+        $this->categoryRepository = $categoryRepository;
+    }
+
+    public function getAllCategories()
```

Comment:
You can remove the word category from the method names, as they are all under the CategoryService. Repeating "category" in the method names does not add any new information.